.travis.yml

```diff
@@ -47,6 +48,7 @@ jobs:
 before_install:
   # Speeds installation of html-proofer
   - export NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+  = gem install bundler -v '< 2.0'
```

Comment:
No, not required.

Reply:
Check the config I mentioned

Reply:
Can you list me out what all changes are required?

Reply:
Just a single change, related to installing of csslint.

Reply:
but wtb bundler then? won't it fail?

Reply:
You can probably just copy the entire `.travis.yml` file from my fork above.

Reply:
No, it won't.